uint8 seq
geometry_msgs/Quaternion orientation
int32[] unknown_data
uint8 battery_soc